service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/CaseManagementOrderIssuedEventHandler.java

```diff
@@ -35,21 +37,24 @@
     private final CaseManagementOrderEmailContentProvider caseManagementOrderEmailContentProvider;
     private final CafcassLookupConfiguration cafcassLookupConfiguration;
     private final IssuedOrderAdminNotificationHandler issuedOrderAdminNotificationHandler;
+    private final DocumentDownloadService documentDownloadService;
 
     @EventListener
     public void sendEmailsForIssuedCaseManagementOrder(final CaseManagementOrderIssuedEvent event) {
         EventData eventData = new EventData(event);
+        CaseManagementOrder issuedCmo = event.getCmo();
 
-        sendToLocalAuthority(eventData);
-        sendToCafcass(eventData);
-        sendToRepresentatives(eventData, DIGITAL_SERVICE);
-        sendToRepresentatives(eventData, EMAIL);
-        issuedOrderAdminNotificationHandler.sendToAdmin(eventData, event.getDocumentContents(), CMO);
+        sendToLocalAuthority(eventData, issuedCmo);
+        sendToCafcass(eventData, issuedCmo);
+        sendToRepresentatives(eventData, issuedCmo, DIGITAL_SERVICE);
+        sendToRepresentatives(eventData, issuedCmo, EMAIL);
+        issuedOrderAdminNotificationHandler.sendToAdmin(eventData,
+            documentDownloadService.downloadDocument(issuedCmo.getOrder().getBinaryUrl()), CMO);
```

Comment:
now we download document 5 times to build link, I have serious concern about performance now

Reply:
https://tools.hmcts.net/jira/browse/FPLA-2061 ticket created